common/api/components-react.api.md

```diff
@@ -2938,14 +2938,8 @@ export interface ToolbarPopupContextProps {
 }
 
 // @public @deprecated
-export interface ToolbarProps extends CommonProps, NoChildrenProps {
-    expandsTo?: Direction;
-    items: CommonToolbarItem[];
-    onItemExecuted?: OnItemExecutedFunc;
-    onKeyDown?: (e: React_2.KeyboardEvent) => void;
-    panelAlignment?: ToolbarPanelAlignment;
-    toolbarOpacitySetting?: ToolbarOpacitySetting;
-    useDragInteraction?: boolean;
+export interface ToolbarProps extends ToolbarWithOverflowProps {
```

Comment:
We are adding `overflowExpandsTo` prop to `ToolbarProps` this way.
It would make more sense to extend `ToolbarWithOverflowProps` from `ToolbarProps`.

Reply:
Seems like a feature addition to the already deprecated API, can't we avoid that?

Reply:
Like I said, we can avoid that in 2 ways, duplicating code, or not using types at all.

If we extend the other way around, we will have an "enableOverflow" props to a component that is only meant to have that to true all the time, which wouldnt make much sense.

Reply:
Reverted all the changes to the components-react component props.